app/scripts/app.js

```diff
@@ -0,0 +1,253 @@
+'use strict';
+/**
+ * @ngdoc overview
+ * @name sbAdminApp
+ * @description
+ * # sbAdminApp
+ *
+ * Main module of the application.
+ */
+
+angular
+  .module('sbAdminApp', [
```

Comment:
lets brand the project from the beginning :-) name the modules after a namespace scheme, e.g. 'ec.main', 'ec.api-access', ... or the long variant 'enviroCar.main'